addons/website/static/src/js/content/menu.js

```diff
@@ -603,7 +607,9 @@ publicWidget.registry.hoverableDropdown = animations.Animation.extend({
         }
         const dropdown = Dropdown.getOrCreateInstance(dropdownToggleEl);
         if (doShow) {
+            this.options.wysiwyg && this.options.wysiwyg.odooEditor.observerUnactive("updateDropdownVisibility");
             dropdown.show();
+            this.options.wysiwyg && this.options.wysiwyg.odooEditor.observerActive("updateDropdownVisibility");
```

Comment:
Yes, I don't like taking too much space as you could have noticed :sweat_smile: Especially for these kinds of lines
Do you think it is correct to do this instead ? :arrow_down: 
```js
this.options.wysiwyg?.odooEditor.observerActive("updateDropdownVisibility");
```
I really don't want to pollute the code with history stuff by doing `if` everytime

Reply:
We're in 17.0, so yes, you can use `?.` (I think you can use it from 16.2 on), and it's a perfect use case here. :)

(Otherwise, in other cases you can just split the line before a `.` for instance.)